### PR TITLE
allow for logging into docker server with username and password

### PR DESCRIPTION
**Pull request by blaw2422**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -41,6 +43,14 @@ func main() {
 
 }
 
+func tryLogin() {
+  if os.Getenv("DOCKER_USERNAME") != "" {
+    if res, err := dockerCmd("login", "-u" ,os.Getenv("DOCKER_USERNAME") ,"-p" ,os.Getenv("DOCKER_PASSWORD") ,"-e" ,os.Getenv("DOCKER_EMAIL") ,os.Getenv("DOCKER_SERVER")); err != nil {
```
Comment (ahmetb) [inline main.go]: `-e` is deprecated and removed:

```
$ docker login gcr.io -u foo -p bar -e zoo
unknown shorthand flag: 'e' in -e
```

Comment (blaw2422) [reply main.go]: i guess we have different versions of docker running.  I didn't have that part in the code at first.  then had to debug why it wasn't working.

I'll make it optional

Comment (blaw2422) [reply main.go]: updated the code.  It will work with both versions now.

**Comment by ahmetb** [comment]
Thanks! I hope the code quality isn't too bad. There might be better tools out there nowadays, I didn't expect this project to be used by others.

**Pull request merged**